src/Marten/Events/Projections/ViewProjection.cs

```diff
@@ -385,14 +390,27 @@ public class ProjectionEvent<T>
         public Guid Id { get; protected set; }
         public int Version { get; protected set; }
         public DateTime Timestamp { get; protected set; }
-        public T Data { get; protected set; }
-
+        public T Data { get; protected set; }
+        public long Sequence { get; protected set; }
+        public Guid StreamId { get; protected set; }
+        public string StreamKey { get; protected set; }
+        public string TenantId { get; protected set; }
+
+        protected void LoadFromEvent(IEvent @event)

```

Comment:
Why not just extend the constructor of `ProjectionEvent` instead of having a separate `LoadFromEvent` method?

Reply:
Rightly, so. I'll do that. Thanks.